android/cli/commands/_buildModule.js

```diff
@@ -33,6 +33,8 @@ const androidDetect = require('../lib/detect').detect,
 	__ = appc.i18n(__dirname).__,
 	version = appc.version;
 
+let tiSdkVersion = '12.6.0.GA';
```

Comment:
Can't this be taken from the package.json / `tiapp` util? Or should it remain this one, even for newer builds?

Reply:
it will be overwritten in `AndroidModuleBuilder`. I thought it might be good to have a default value

Reply:
but let me check if I can move that right to the initialization